pkg/image/validation/os_test.go

```diff
@@ -677,3 +677,68 @@ func TestValidateRawConfiguration(t *testing.T) {
 		})
 	}
 }
+
+func TestValidateTimeSync(t *testing.T) {
+	tests := map[string]struct {
+		Definition             image.Definition
+		ExpectedFailedMessages []string
+	}{
+		`not included`: {
+			Definition: image.Definition{},
+		},
+		`forceWait specified and NTP sources valid`: {
```

Comment:
I'd suggest to split the test case into two:
- one where only pools are defined
- one where only servers are defined

Reply:
OK will do.